### ISSUE TITLE
:children_crossing: Provide a copy button to facilitate link sharing

### ISSUE THREAD

**Issue by mitchdawson1982** [ministryofjustice/find-moj-data#1088]
### User Story

As a user
I expect to be able to easily copy page url links to a data asset
So that I can share them with colleagues

### Value / Purpose

A quick copy link / button is common place on modern websites to aid usability, providing one on the front end will help users to quickly and easily copy links to assets to aid sharing.

### Hypothesis

If we... [do a thing]
Then... [this will happen]

### Definition of Done

Example - [ ] Documentation has been written / updated
- [ ] README has been updated
- [ ] User docs have been updated
- [ ] Another team member has reviewed
- [ ] Tests are green

**Comment by murdo-moj** [comment]
https://github.com/ministryofjustice/find-moj-data/pull/1092

**Comment by murdo-moj** [comment]
![Image](https://github.com/user-attachments/assets/f4ad3b0e-fd70-4e59-8016-528e3cc15c76)

**Comment by YvanMOJdigital** [comment]
for design input

**Comment by joe-horton-moj** [comment]
Hi team, I've recorded a short loom to discuss this ticket and the 2 options we could do.

**Loom video**

https://www.loom.com/share/6dcc9c8dc9374d1c951290318d534408?sid=d11f335a-2595-47f1-af3b-680e3cd592d3

**TL;DR**

1. my first recommendation is that I don't think we need it, most browsers have the functionality to quickly copy the URL so adding additional functionality into our service without a strong user need wouldn't be ideal in my opinion
2. if there is a general consensus that it is needed and would be valued, then I've mocked up a version (shown below).

**Design for option 2**

<img width="1438" alt="Image" src="https://github.com/user-attachments/assets/e2a4eef8-bf16-4994-8107-f30433167002" />

- [ ] group page CTA's
- [ ] align group horizontally using MoJ design system as an example, below the H1 of the page
- [ ] make both CTA's secondary
- [ ] "Copy URL" as the label instead of "Copy link"

**Comment by joe-horton-moj** [comment]
Had a chat with @murdo-moj and we both agreed 'copy url' action isn't needed. 

Leave design as is (just leave the feedback link button where it is and not group page action buttons under the title like in DBT's data workspace.)
